### ISSUE TITLE
Email the user when the game account is made

### ISSUE THREAD

**Issue by nicholaslum444** [NUSComputingDev/gfp#4]

**Comment by Skaty** [comment]
Do you have a specific template in mind?

**Comment by Skaty** [comment]
@nicholaslum444 & @jmtiong any response on that?

**Comment by nicholaslum444** [comment]
Sorry, missed the message.

Sample below. @jmtiong Please expand on this.

````
Hi [STUDENT], 

Thanks for signing up for GFP 2016. As there will be games and fun activities on the night itself, please read the following instructions before the event!

Instructions:
1. [INSTRUCTION NUMBER ONE]
2. [INSTRUCTION NUMBER TWO]
3. ....
...
X-1. [INSTRUCTION NUMBER X-1]
X. Finally, please click the following link to confirm your game account for GFP 2016! http://gfp.nuscomputing.com/signup/confirm?user=8276191&secret=8iq78yy78eu

Warmest Regards,
GFP Team
````

For the link, I'm envisioning a simple GET request, i.e. when you sign up for a new Facebook account and they want you to verify your email, you must click a certain link that will verify the account. On the server side after visiting this link, the db should have a field updated for the user that shows he has "activated" his account. This also serves a dual purpose of him acknowledging receipt of the email.

@jmtiong Please evaluate if this feature is helpful. 

@Skaty Please consider the difficulty of this feature and feedback if there is time constraint. 

Thanks

**Comment by jmtiong** [comment]
Yes this feature is a must have. In addition, the email will also contain a randomly generated password for them and they would have to change their password upon first login

**Comment by Skaty** [comment]
Who would log in? Both the participants and game master?
